package-lock.json

```diff
@@ -0,0 +1,1935 @@
+{
+  "name": "cmake-tools",
```

Comment:
Please delete `package.lock`.  This project uses yarn.lock instead.  Maybe it would be worth adding `package.lock` to `.gitignore` to avoid accidentally submitting this file in the future.